### PR TITLE
Work/rhodh/ways of working

### PR DESCRIPTION
**Pull request by rhodh**
Setup Pr for group review

### CODE REVIEW

**Comment by KarenFarrell36** [comment]
Initial comments - will keep thinking..
Collaboration
* Knowledge Sharing
Continuous Improvement
* Lessons Learned Sessions
* Agree with retro comment but this analysis should not be restricted to the one ceremony
Delivery
* Collaboration with Stakeholders to solidify understanding of requirements
* Prioritising workload and re-planning
* Suggest improvements that add value to Stakeholders
Champion Quality
* Bug fixes improve our regression scenario coverage too (at least will do)
* Consider potential risks and mitigate where possible

**Comment by dingjimmy** [comment]
Some thought to add around collaboration: 

- Treat each other with Respect [Respect is key to building Trust and trust is essential to effective collaboration]
- Assume noble intent. [Misinterpretations & Misunderstandings happen, but most of the time people have good intentions.]

File: docs/6.-Engineering/Ways-Of-Working/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+next-review: 
+---
+# Ways of Working
+
+While we encourage empowered self organizing teams, there are some default ways of working teams are expected to adopt. These can be broken down into 5 Pillars
+
+ - Collaboration
+ - Test First Delivery
+ - Driving Delivery 
+ - Champion Quality (TBD name)
```
Comment (91dave) [inline docs/6.-Engineering/Ways-Of-Working/README.md:16]: How about **Ownership of Delivery** and **Quality Emphasis** to make these both nouns like the others?

On reflection I dislike "pillars" and would suggest **core principles**, but this is very much personal preferance!

Comment (rhodh) [reply docs/6.-Engineering/Ways-Of-Working/README.md:16]: Making terms consistent is a good idea. 

I think I prefer pillars to core principles just to try and emphasise the idea that these are building blocks to empowered autonomous teams but both work tbh. 

@KarenFarrell36 @dingjimmy @beeozguler which do prefer?

**Comment by beeozguler** [comment]
**Initial notes from me:**

I like the title Champion Quality, I don't think we should change it
Did we say something about Knowledge Sharing? It fits under every point anyway but could make a bigger point of it

Collaboration: 
-	Use tools that encourage you to be collaborative (miro?)
-	Ask questions and participate in conversations

Test first delivery:
-	Understand the importance of testing 
-	Don’t put all the pressure of testing on one person – everyone is responsible for a good product 

Own Product Delivery:
-	Ask your PM questions
-	Familiarise yourself with the stakeholders

Continuous Improvement: 
-	Keep learning, training, and sharing your knowledge

**Comment by beeozguler** [comment]
> Some thought to add around collaboration:
> 
> * Treat each other with Respect [Respect is key to building Trust and trust is essential to effective collaboration]
> * Assume noble intent. [Misinterpretations & Misunderstandings happen, but most of the time people have good intentions.]

these ones are great

**Comment by KarenFarrell36** [comment]
Couple more comments but happy to be challenged (English is certainly not my strong point)

### Ways of Working
The following will summarize **~~the~~ what** each Pillar represents and give non exhaustive list sharing examples of good practices for each pillar.
**Also, Pillar or pillar ??**

### Collaboration
Empowered **self-organizing** teams are expected to be proactive**~~ly~~** in their collaboration. Sharing information widely and loudly. On the lookout for opportunities to code together and consequently share knowledge and gain fast feedback on work.

### Champion Quality
Choosing the correct technology to solve the problem, which doesn't always means our sting to our defaults.

Think outside the box .... how can we improve quality if had no boundaries

### Continuous Improvement
Remember that **a** team **~~we~~** can't evolve if they aren't looking at way**s** to continuously improve **~~the team~~ themselves** and the platform. This doesn't mean they'll always be successful - we can learn more from our failures than our successes as long as we're willing to be honest and self-critical with each other.

Recognize pain point**s** and come up with active plans to resolve them (don't just leave it to the next sprint ceremony).

**American spelling throughout – is this ok?**

**Comment by KarenFarrell36** [comment]
Following a recent conversation, we should also mention being conscious of clear communication, especially with our International colleagues e.g. do not use idioms etc

**Comment by 91dave** [comment]
Is it worth linking this out to the three ways of DevOps? One specific point that I feel would be worth calling out is reducing WIP whenever possible (and thus completing things earlier and increasing flow).

Does this warrant its own section/pillar, or is it potentially a bullet point in the Collaboration section?

@rhodh / @beeozguler / @KarenFarrell36 / @dingjimmy

**Pull request closed without merging**